sources/packages/backend/apps/api/src/services/application-restriction-bypass/application-restriction-bypass.service.ts

```diff
@@ -17,33 +15,35 @@ export class ApplicationRestrictionBypassService extends RecordDataModelService<
   }
 
   /**
-   * Returns all restriction bypasses for a given application.
+   * Returns all application restriction bypasses for a given application.
    * @param applicationId id of the application to retrieve restriction bypasses.
    * @returns application restriction bypasses.
    */
   async getApplicationRestrictionBypasses(
     applicationId: number,
-  ): Promise<ApplicationRestrictionBypassSummaryAPIOutDTO[]> {
-    return this.repo
-      .createQueryBuilder("applicationRestrictionBypass")
-      .select("applicationRestrictionBypass.id", "id")
-      .addSelect("applicationRestrictionBypass.createdAt", "createdAt")
-      .addSelect("restriction.restrictionType", "restrictionType")
-      .addSelect("restriction.restrictionCode", "restrictionCode")
-      .addSelect("studentRestriction.isActive", "isActive")
-      .innerJoin(
-        "applicationRestrictionBypass.studentRestriction",
-        "studentRestriction",
-      )
-      .innerJoin("studentRestriction.restriction", "restriction")
-      .innerJoin("applicationRestrictionBypass.application", "application")
-      .where("application.id = :applicationId", {
-        applicationId,
-      })
-      .andWhere("application.applicationStatus != :applicationStatus", {
-        applicationStatus: ApplicationStatus.Draft,
-      })
-      .orderBy("applicationRestrictionBypass.createdAt", "DESC")
-      .getRawMany<ApplicationRestrictionBypassSummaryAPIOutDTO>();
+  ): Promise<ApplicationRestrictionBypass[]> {
+    return this.repo.find({
+      select: {
+        id: true,
+        application: { id: true },
```

Comment:
`isActive` for the bypass is missing here.

Reply:
Updated just now.